react/features/welcome/components/AbstractWelcomePage.js

```diff
@@ -13,11 +14,13 @@ import { generateRoomWithoutSeparator } from '../functions';
  */
 type Props = {
 
+    _room: string,
```

Comment:
jsdoc is missing

Reply:
Absolutely, so as for the dispatch :) But that's not new code, I just relocated it. Anyhow, I add it, as it's an easy fix.